test/mpi/session/testlist

```diff
@@ -5,3 +5,4 @@ session_re_init 4
 session_psets 1
 session_self 1
 session_bsend 2
+session_non_coll 2 timeLimit=10 arg=: arg=-n arg=1 arg=true
```

Comment:
this might be a bit too clever but i can have `mpiexec` launch an extra process by passing these "args". the resulting world size will be 3, and only 2 procs will call session init.

Reply:
The reason I abandoned my even/odd pid approach for now is that I won't know ahead of time which process to `printf("No Errors\n")` from. Alternatively we can use a different test runner that does not look for that output, but I'm not sure if that is still an option or if I have to add it.

Reply:
done